ruby_exercises/mythical-creatures/lib/hobbit.rb

```diff
@@ -0,0 +1,32 @@
+class Hobbit
+  attr_reader :name,
+              :disposition,
+              :age
+
+  def initialize(name, disposition = 'homebody')
+    @name = name
+    @disposition = disposition
+    @age = 0
+  end
+
+  def celebrate_birthday
+    @age += 1
+  end
+
+  def adult?
+     @age >= 33
+  end
+
+  def old?
+    @age >= 101
+  end
+
+  def has_ring?(name)
+    name == 'Frodo'
+  end
+
+  def is_short?
```

Comment:
This totally works.
Another way to think about this is that we initialize an object with attributes. Being short is an attribute of a Hobbit, so I'd be tempted to make a @short = true instance variable at initialization and then have `is_short?` return the value of @short.